tests/utils/session.py

```diff
@@ -77,26 +84,26 @@ def delete_resource(self, path: str, **kwargs) -> dict:
         return self.checked_delete(path, **kwargs)
 
     def checked_get(self, path: str, **kwargs) -> dict:
-        self.calls.append(FakeCall('GET', path, params=kwargs.get('params')))
+        self.calls.append(FakeCall('GET', path, params=kwargs.get('params'), version=kwargs.get('version')))
```

Comment:
At this point, it almost makes more sense to pass kwargs into the FakeCall constructor.